content/gungier/index.md

```diff
@@ -0,0 +1,19 @@
+---
+{
+	name: "Gabriel Ungier",
+	firstName: "Gabriel",
+	lastName: "Ungier",
+	description: "webdev, JavaScript/Typescript, world traveler, avid reader, loves writing code and hopefully one day will be good at it, too!",
+	socials: {
+		github: "gungier",
+        x: "@GabrielUngier",
+		website: "re-coming-again-soon-one-day",
```

Comment:
```suggestion
		twitter: "@GabrielUngier",
```

Tiny fix for these links - our pages don't look for an `x` property yet.
& `website` needs to be removed if it isn't a URL, otherwise the page will still try to link to it :)

Reply:
Ah, was trying to be funny with the website, since it's not up anymore.  And, I can switch it to twitter np.